pkg/fixtures/fixtures.go

```diff
@@ -454,6 +447,30 @@ func (fxt *Fixture) parseQuery(value string) (string, error) {
 	return value, nil
 }
 
+func getEnvVar(query fixtureQuery) (string, error) {
```

Comment:
I think this definitely makes sense to be it's own function. I can see it being used elsewhere eventually.